.nvmrc

```
@@ -0,0 +1 @@
+12.7.0
```

We should use the release label
```suggestion
lts/erbium
```